src/NLog/Targets/TraceTarget.cs

```diff
@@ -62,6 +63,12 @@ namespace NLog.Targets
     public sealed class TraceTarget : TargetWithLayout
     {
         /// <summary>
+        /// Always use <see cref="Trace.WriteLine(string)"/> independent of <see cref="LogLevel"/>
+        /// </summary>
+        [DefaultValue(false)]
+        public bool WriteLine { get; set; }
```

Comment:
Not sure why you like "prepend" and "prefix". It is just the Visual Studio Listener that performs some prefixing, it could just as well add suffix depending on LogLevel provided to the Trace-call.